scripts/lb-wrapper

```diff
@@ -60,8 +62,11 @@ log "Info" "Running xklb command: ${xklb_full_cmd}"
 # >(...) "process substitution" explained at https://unix.stackexchange.com/a/324170
 # 1>&2 redirect back-to-STDERR to avoid nested (repeat) logging, explained at https://stackoverflow.com/a/15936384
 eval "${xklb_full_cmd}" \
-    > >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]]; then echo "$line"; else log "Info" "$line"; fi; done) \
-    2> >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]]; then echo "$line"; else log "Debug" "$line" 1>&2; fi; done) &
+    # > >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]]; then echo "$line"; else log "Info" "$line"; fi; done) \
+    # 2> >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]]; then echo "$line"; else log "Debug" "$line" 1>&2; fi; done) &
+    # adjust above line to also reroute the return code of the underlying command as a line saying "return code: 0" or "return code: 1" or "return code: 2" etc.
+    > >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]] || [[ $line == "yt_dlp_rc: "* ]]; then echo "$line"; else log "Info" "$line"; fi; done) \
+    2> >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]]; then echo "$line"; else log "Debug" "$line" 1>&2; fi; done; echo "yt_dlp_rc: $?" 1>&2) &
```

Comment:
I'm trying to understand this.

@deldesir is the following closer to the truth, of what we're trying to accomplish?

```suggestion
    # adjust above line to also reroute the return code of the underlying command as a line saying "xklb_rc: 0" or "xklb_rc: 1" or "xklb_rc: 2" etc.
    > >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* || $line == "xklb_rc: "* ]]; then echo "$line"; else log "Info" "$line"; fi; done) \
    2> >(while read -r line; do if [[ $line == downloading* || $line =~ \[https://.*\]:* ]]; then echo "$line"; else log "Debug" "$line" 1>&2; fi; done; echo "xklb_rc: $?" 1>&2) &
```

Reply:
I was trying to make lb-wrapper to report underlying yt-dlp return code. We already had $rc for xklb.

Reply:
It's the exit status of the last underlying command (yt-dlp) executed by xklb.

Reply:
I could see it's yt-dlp because I could reroute its output. This exit status code is not doing anything useful here. It was me debugging yt-dlp and forgot to remove it.

Reply:
> This exit status code is not doing anything useful here. It was me debugging yt-dlp and I forgot to remove this code.

1. Please remove any temporary code and confirm.

2. Please also fix Lines 56, 58 and 60 which contain (former variable) XKLB_EXECUTABLE.

Thanks!

Reply:
Temporary code removed.